suite-native/toast-notifications/src/slice.ts

```diff
@@ -0,0 +1,48 @@
+import { createSlice, PayloadAction, Slice } from '@reduxjs/toolkit';
+import { A } from '@mobily/ts-belt';
+
+import {
+    ToastNotificationsRootState,
```

Comment:
Please define root state type in this file

Reply:
just a convention across slices and reducers to have type in same file. Nothing too critical, same thing as you want to see type props in component

Reply:
fixed in 148b2cb690ef72b5a2c4628595be1d22506c4b48